staging/src/k8s.io/apiserver/go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
 	github.com/pquerna/cachecontrol v0.0.0-20171018203845-0dec1b30a021 // indirect
+	github.com/sirupsen/logrus v1.8.1 // indirect
```

Comment:
suspect `hack/lint-dependencies.sh` will turn up something for this.

Reply:
or not? https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/103743/pull-kubernetes-dependencies/1416124150098956288 seems happy.